frontend/app/blueprints/sample/templates/sample.html

```diff
@@ -110,17 +110,23 @@ <h6 class="card-title">
 
 
 {% macro similar_samples_card(data) %}
-<div class="card col-md-auto {% if data is none %}border-warning{% endif %}" style="width: 24rem;">
-    <div class="card-header">Similar samples {% if data is not none %}[<i>{{ data.typing_method }}</i>]{% endif %}</div>
-    <div class="card-body">
-        {% if data is none %}
-            <p class="card-text">
-                Failed to find samples. 
-                Please contact the administrator.
-            </p>
-        {% else %}
-            <div id="tree-body" style="pointer-events: none;"></div>
-        {% endif %}
+<div id="similar-samples-card"
+     class="card col-md-auto {% if data is none %}border-warning{% endif %}" 
+     style="width: 24rem;">
```

Comment:
Could the styling constants be extracted into a single document? That makes it much easier to keep the styling consistent in the application, and also to make updates in the future